### ISSUE TITLE
Check datadir compatibility with latest geoserver 2.22

### ISSUE THREAD

**Issue by fvanderbiest** [georchestra/geoserver_minimal_datadir#17]
We should bootstrap a vanilla geoserver 2.22 instance from an empty or truly minimal datadir (eg https://github.com/geoserver/geoserver/tree/main/data/minimal ) and check the files we provide here are still up to date, using eg meld.

**Comment by landryb** [comment]
for one thing at least.. logging. starting a gs 2.22.2 creates XML files under `logs`:
```
DEFAULT_LOGGING.xml
GEOSERVER_DEVELOPER_LOGGING.xml
GEOTOOLS_DEVELOPER_LOGGING.xml
PRODUCTION_LOGGING.xml
QUIET_LOGGING.xml
TEST_LOGGING.xml
VERBOSE_LOGGING.xml
```
we ship .properties in https://github.com/georchestra/geoserver_minimal_datadir/tree/master/logs, and afaict gs only uses the xml ones:
```
02 Jan 15:51:10 INFO   [geoserver.logging] - Beginning GeoServer cleanup sequence
02 Jan 15:51:26 CONFIG [geoserver.logging] - Log4j 2 configuration set to DEFAULT_LOGGING.xml
```
in the webui, going to the global tab, the profiles listed are the xml ones:
![image](https://github.com/georchestra/geoserver_minimal_datadir/assets/3421760/5a4bdefb-41c4-4648-a721-8ea2fc1a2849)

**Comment by landryb** [comment]
and afaict that's since the update to log4j2 in 2.21 per https://docs.geoserver.org/2.22.x/en/user/installation/upgrade.html#log4j-upgrade-geoserver-2-21-and-newer

**Comment by landryb** [comment]
more or less fixed by #19
